coalib/settings/Setting.py

```diff
@@ -108,7 +125,7 @@ def __init__(self,
         self.key = key
         self.origin = str(origin)
 
-    def __path__(self, origin=None):
+    def __path__(self, origin=None, globbed_origin=False):
```

Comment:
I agree the name should be changed, but since the boolean basically tells the function to whether or not escape `origin`, I think the name should be `glob_escape_origin` as it's more explicit (plus `is_glob` feels like an assertion test more than a boolean setting). What do you think? @Makman2